Description: Switch build process to Kbuild instead of Makefile
 Switch build process to Kbuild instead of Makefile
Author: Werner Sembach <tux@tuxedocomputers.com>

---

--- /dev/null
+++ tuxedo-yt6801-1.0.28/Kbuild
@@ -0,0 +1,2 @@
+yt6801-objs :=  fuxi-gmac-common.o fuxi-gmac-desc.o fuxi-gmac-ethtool.o fuxi-gmac-hw.o fuxi-gmac-net.o fuxi-gmac-pci.o fuxi-gmac-phy.o fuxi-efuse.o fuxi-dbg.o fuxi-gmac-debugfs.o
+obj-m += yt6801.o
